config/docker/image/app-volumes/Dockerfile

```diff
@@ -4,10 +4,10 @@ RUN mkdir -p /var/www/html/vendor \
     /var/www/html/generated \
     /var/www/html/var \
     /var/www/html/pub/static \
-    /var/www/html/pub/media
+    /var/www/html/pub/media \
+    /var/www/html/dev/tests/integration/tmp \
+    && chown -R 1000:1000 /var/www/html
```

Comment:
Could you please test that this sets the right permissions even if not doing the `chown` for the specific subfolders? Thanks.

Reply:
It's working fine! 👍